travis-scripts/script.sh

```diff
@@ -39,7 +39,7 @@ then
     make CFLAGS=-Werror
 elif [ "$JOB_TYPE" = compile_and_unit_test ]
 then
-    make CFLAGS=-Werror  &&
+    make CFLAGS="-Werror -Wno-pointer-sign -Wno-duplicate-decl-specifier"  &&
```

Comment:
Can you add an explanation to the commit message why it's OK to skip these two? Or maybe some TODO comment in the travis file to fix that in the future?

Reply:
Thanks, added.